src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/custom-timing-details.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Use $har for custom timing details
+tags:
+    - Synthetics
+    - Synthetic monitoring
+    - Scripting monitors
+    - Scripted API monitor
+metaDescription: You can now use $har to have your scripted API monitors report custom timing details that appear in the New Relic UI.
```

Comment:
I think this might be for line 13 (You need the new runtime) and I agree. Maybe we should use something like this instead:

The Node 16.10 runtime or newer (link to the transition guide) to create custom timing details for scripted API monitors.